src/_utils/item/Item.tsx

```diff
@@ -19,7 +19,7 @@ export interface ItemProps extends A11yProps {
   readonly href?: string | JSX.Element
   readonly highlighted?: boolean
   readonly isClickable?: boolean
-  readonly leftTitle?: string
+  readonly leftTitle?: React.ReactNode
```

Comment:
I guess it should be: `string | React.ReactNode` here

Reply:
- [ ] Update type?

Reply:
No need. React.RectNode has intrinsic `string, number, node`...

Reply:
ReactNode has string, number, boolean intrisically